include/z64.h

```diff
@@ -395,11 +395,11 @@ typedef struct {
     /* 0x188 */ Vtx* questPageVtx;
     /* 0x18C */ Vtx* maskPageVtx;
     /* 0x190 */ Vtx* itemVtx;
-    /* 0x194 */ Vtx* mapVtx;
+    /* 0x194 */ UNK_TYPE1 unk_194;
```

Comment:
```suggestion
    /* 0x194 */ UNK_TYPE1 unk_194[0x4];
```
To fill up the space